app/schemas/models/read_announcement.schema.js

```diff
@@ -0,0 +1,37 @@
+let ReadAnnouncementSchema;
+const c = require('./../schemas');
+
+module.exports =
+  (ReadAnnouncementSchema = {
```

Comment:
Do we need the auto-translated-from-CoffeeScript-style weird variable declaration stuff here?